src/Microsoft.CSharp/ref/project.json

```diff
@@ -1,14 +1,10 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Dynamic.Runtime": "4.0.0",
-    "System.Linq.Expressions": "4.0.0"
+    "System.Runtime": "4.1.0",
+    "System.Dynamic.Runtime": "4.0.11",
+    "System.Linq.Expressions": "4.1.0"
```

Comment:
I guess since you intend to make these depend on new API we can update all of them.